src/app/credential-issuer/middleware.ts

```diff
@@ -44,7 +44,7 @@ export const sendParamsToAPI: RequestHandler = async (req, res) => {
   }
 
   const apiResponse = await postCriCallback(req, body);
-  return handleBackendResponse(req, res, apiResponse?.data);
+  return handleBackendResponse(req, res, apiResponse.data);
```

Comment:
data. isn't an optional parameter on AxiosResponse, the return type of postCriCallback